server/server.go

```diff
@@ -99,6 +101,9 @@ func Start(opt Opt) error {
 			return err
 		}
 		logger = logger.WithField("ssh-addr", sshln.Addr())
+		if opt.ProxyProtocol {
+			sshln = &proxyproto.Listener{Listener: sshln}
+		}
```

Comment:
We could similarly wrap the `wsln` listener below, although in that case there are multiple methods in which the client's real IP address can reported (`X-Forwarded-For` and `X-real-ip` headers for example). So I excluded it as out of the scope for this PR.

Reply:
Sounds good. I'm thinking by default for the `wsln` it will automatically uses IPs from `X-Forwarded-For` if the value exists. 

I also renamed the flag from `--proxy-protocol` to `--ssh-proxy-protocol` to be explicit because it only makes sense to enable the PROXY protocol for the ssh listener. I don't expect people would need this for the `wsln`.